rcon/broadcast.py

```diff
@@ -1,23 +1,36 @@
 import os
 import time
+import random
 
 from rcon.extended_commands import Rcon
 from rcon.settings import SERVER_INFO
+from rcon.models import init_db, enter_session
+from rcon.user_config import get_user_config, set_user_config
 
 if __name__ == '__main__':
     ctl = Rcon(
         SERVER_INFO
     )
     mount_path = os.getenv("BROADCAST_PATH", '')
+
+    set_user_config('randomize_broadcast_available', False)
     
-    with open(f"{mount_path}broadcasts.txt") as f: 
-        msgs = [l.split(' ', 1) for l in f]
-        msgs = [m for m in msgs if len(m) == 2] # filtering out badly formated lines
- 
-    while True: 
-        for time_sec, msg in msgs:
-            if msg == '/nextmap\n':
-                msg = "Next map: {}".format(ctl.get_next_map())
-            print(f"{time_sec} second(s): {msg}")
-            ctl.set_broadcast(msg) 
-            time.sleep(int(time_sec)) 
+    if os.path.isfile(f"{mount_path}broadcasts.txt"):
+        with open(f"{mount_path}broadcasts.txt") as f: 
+            default_msgs = [l.split(' ', 1) for l in f]
+            default_msgs = [m for m in default_msgs if len(m) == 2] # filtering out badly formated lines
+
+        if type(default_msgs) is list:
+            set_user_config('randomize_broadcast_available', True)
```

Comment:
I wrote some code abstracting the config of the broadcast and taking care of default in `rcon/user_config.py` so we should remove this one before merging.